src/ert/gui/simulation/view/realization.py

```diff
@@ -153,3 +170,25 @@ def _paint_inner_grid(painter: QPainter, rect: QRect, colors) -> None:
 
     def sizeHint(self, option, index) -> QSize:
         return self._size
+
+    def eventFilter(self, watched, event: QEvent):
+        if event.type() == QEvent.Type.ToolTip:
+            mouse_pos = event.pos()
+            parent: RealizationWidget = self.parent()
+            view = parent._real_view
+            model: QAbstractItemModel = view.model()
+            for row in range(model.rowCount()):
+                index = model.index(row, 0)
+                rect = view.visualRect(index)
+                if rect.contains(mouse_pos):
+                    mem_msg = index.data(MemoryUsageRole)
+                    if mem_msg:
+                        (current_memory_usage, maximum_memory_usage) = mem_msg
+                        txt = (
+                            f"Current memory usage: {byte_with_unit(current_memory_usage)}\n"
+                            f"Maximum memory usage: {byte_with_unit(maximum_memory_usage)}"
```

Comment:
Looks good, but have you played around with justification etc to see if you could align the lines better?
Either justifying or with tabs or something?
How does this look when using RGS onprem?

Reply:
This is the best I got the alignment. It might be a different font on mac, because the alignment looked better there. 
![image](https://github.com/equinor/ert/assets/107626001/c2b04c9f-3ccd-4cb4-8cf7-046a2233add2)

Reply:
I tried right aligning the text instead (not with the amount/unit), but I found that left alignment looked better.

Reply:
This is the same, but with right alignment. Right alignment looked better on mac, and left alignment looked better on rgs node.
![image](https://github.com/equinor/ert/assets/107626001/f4b097c1-b213-44d9-94cf-c8ea0cec15ef)

Reply:
I think the formatting you have now is pretty close to perfect.
What bugs me a bit is that the tooltip seems to be handled incorrectly, in the way that sometimes you get it, and sometimes you don't. Could there be more events taking place?

Reply:
Yes, it seems to be VERY flaky and random. I will ask chatgpt, maybe he knows a better way.

Reply:
At least check how many times the hover-thing happens. It might be that you get multiple hits, or some with invalid indices.